sdk/include/psdk/ntddndis.h

```diff
@@ -331,6 +343,37 @@ typedef ULONG NDIS_OID, *PNDIS_OID;
 #define OID_PNP_WAKE_UP_OK                      0xFD020200
 #define OID_PNP_WAKE_UP_ERROR                   0xFD020201
 
+/* TCP and IP OIDs */
+#define OID_TCP_TASK_OFFLOAD                    0xFC010201
+#define OID_TCP_TASK_IPSEC_ADD_SA               0xFC010202
+#define OID_TCP_TASK_IPSEC_DELETE_SA            0xFC010203
+#define OID_TCP_SAN_SUPPORT                     0xFC010204
+#define OID_TCP_TASK_IPSEC_ADD_UDPESP_SA        0xFC010205
+#define OID_TCP_TASK_IPSEC_DELETE_UDPESP_SA     0xFC010206
+#define OID_TCP4_OFFLOAD_STATS                  0xFC010207
+#define OID_TCP6_OFFLOAD_STATS                  0xFC010208
+#define OID_IP4_OFFLOAD_STATS                   0xFC010209
+#define OID_IP6_OFFLOAD_STATS                   0xFC01020A
+
+/* New NDIS 6 offload OIDs */
+#define OID_TCP_OFFLOAD_CURRENT_CONFIG                     0xFC01020B /* NDIS 5 handled. Query only */
+#define OID_TCP_OFFLOAD_PARAMETERS                         0xFC01020C /* Set only */
+#define OID_TCP_OFFLOAD_HARDWARE_CAPABILITIES              0xFC01020D /* Query only */
+#define OID_TCP_CONNECTION_OFFLOAD_CURRENT_CONFIG          0xFC01020E /* Query only */
+#define OID_TCP_CONNECTION_OFFLOAD_HARDWARE_CAPABILITIES   0xFC01020F /* Query only */
+#define OID_OFFLOAD_ENCAPSULATION                          0x0101010A
+
+
```

Comment:
Unless the rest of the file already does this, I'd stick with a single empty line.